src/studio/src/designer/backend/Designer.csproj

```diff
@@ -24,7 +24,7 @@
     <PackageReference Include="ini-parser-netstandard" Version="2.5.2" />
     <PackageReference Include="JWTCookieAuthentication" Version="2.4.1-alpha" />
     <!-- Do not upgrade this package unless extensively tested-->
-    <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0034" />
+    <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0100" />
```

Comment:
Changes of libgit2sharp dependencies should be in a seperate PR as we always experience issues with this.